### ISSUE TITLE
Consider other filters than moving average

### ISSUE THREAD

**Issue by jbouzanquet** [CSULA-Qdots/filter#12]
I was just reading an [article on climate change](http://arstechnica.com/science/2013/02/understanding-extreme-weather-in-an-era-of-climate-change/#image-2), and one of the [graphs in the article](http://cdn.arstechnica.net/wp-content/uploads/2013/02/brief-extreme-weather-figure-2.png) intrigued me. They show a curve rather like we want to produce for our "filtered" or "smoothed" data, but use what they call a binomial filter. Google produces a [page with a simple description of this filter](http://folk.uib.no/ngbnk/kurs/notes/node107.html). The binomial filter appears to be a moving average weighted by binomial coefficients. Is it worth trying it instead of the moving average?

**Comment by blackdjinn** [comment]
Yes. We should consider them. These are doing something different in terms of how they transform the data, however. These behave by taking a noisy signal and returning a less noisy signal, so the original data points are replaced. The current filter is designed to 'cherry pick' the data by selecting points that are less noise-like. No real problem with either method, tho.

**Comment by jbouzanquet** [comment]
Something that also came to mind is if we use a binomial filter with the weights reflected across the axes 1/4 way from the center. Basically the far edges count more. I'm not sure I'm thinking through that right, particularly with my headache right now. I'm trying to come up with a way to eliminate the spikes. See the spreadsheets I made from generated data that still have some of the spikes remaining.

**Comment by blackdjinn** [comment]
No... the binomial puts more emphasis on the local points.

Think about it this way:
1. Run a Fourier transform on the data.
2. Throw away all but the lowest frequency components.
3. Plot the resulting function.

The results should be a LOT closer to the actual signal than the noisy ones we have.
The whole running-average or binomial methods are ways of stripping the high-frequency noise components out of the data signal.
